crates/aiken-lang/src/ast.rs

```diff
@@ -258,6 +258,18 @@ pub struct ModuleConstant<T, ConstantRecordTag> {
     pub tipo: T,
 }
 
+pub type TypedValidator = Validator<Arc<Type>, TypedExpr>;
+pub type UntypedValidator = Validator<(), UntypedExpr>;
+
+#[derive(Debug, Clone, PartialEq)]
+pub struct Validator<T, Expr> {
+    pub doc: Option<String>,
+    pub end_position: usize,
+    pub fun: Function<T, Expr>,
```

Comment:
This means that at the moment, we can only have one function per Validator. Which, I am okay with. Long-term, we probably want to allow grouping multiple definitions into one.

Reply:
it doesn't mean that